pkg/platform/abstract/platform.go

```diff
@@ -638,36 +642,6 @@ func (ap *Platform) GetRuntimeBuildArgs(runtime runtime.Runtime) map[string]stri
 	return runtime.GetRuntimeBuildArgs(ap.Config.Runtime)
 }
 
-func (ap *Platform) functionBuildRequired(functionConfig *functionconfig.Config) (bool, error) {
```

Comment:
Note for reviewer - this function wasn't deleted, just moved lower to have all Public functions above Private functions